mbuild/compound.py

```diff
@@ -69,6 +72,17 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
     compound : mb.Compound
 
     """
+    # First check if we are loading from an existing parmed or trajectory structure
+    if structure:
```

Comment:
What necessitated the addition of the `structure` kwarg? Could you not just check if `filename` is an instance and do the appropriate thing based on that?

If that's possible, `filename` might be better called something like `filename_or_topology`

Reply:
You are right, the `structure` kwarg seems redundant here. I will remove it in the next commit.